src/runner/Resources.resx

```diff
@@ -160,5 +160,12 @@
   <data name="EXIT_MENU_TEXT" xml:space="preserve">
     <value>Exit</value>
     <comment>Exit as a verb, as in Exit the application</comment>
-  </data>  
+  </data>
+  <data name="GITHUB_NEW_VERSION_USING_LOCAL_BUILD_ERROR" xml:space="preserve">
+    <value>Updating from a local build is not supported.</value>
+    <comment>User cannot autoupdate from a locally-built PowerToys version</comment>
```

Comment:
"Updating from a local build is not possible" sounds better

Reply:
or "Autoupdating"

Reply:
It is possible, but not supported ;)

Reply:
This is only for devs, not for the regular users that will never see that message.

Reply:
then, "is not supported"

Reply:
> This is only for devs, not for the regular users that will never see that message.

But still...

Reply:
"Not possible" is technically wrong. Since this is a message for devs, it has to be technically precise.

Reply:
> then, "is not supported"

I already admitted that ;)